Makefile

```diff
@@ -783,6 +783,16 @@ backup-db: ## Backup the gis database
 	@cp backups/osgisstack-gis-database.dmp backups/osgisstack-gis-database-$$(date +%Y-%m-%d).dmp
 	@ls -lah backups/osgisstack-gis-database*
 
+restore-db: ## Restore the gis database from a back up
+	@make check-env
+	@echo
+	@echo "------------------------------------------------------------------"
+	@echo "Restoring the entire GIS postgres db from a backup"
+	@echo "------------------------------------------------------------------"
+	@docker cp backups/osgisstack-gis-database.dmp osgisstack_db_1:/tmp/
+	@COMPOSE_PROFILES=$(shell paste -sd, enabled-profiles) docker-compose exec -u postgres db psql -c "DROP DATABASE IF EXISTS gis WITH (FORCE);"
```

Comment:
If we do something destructive like this we should prompt the user to check if they are really sure. Look in the makefile for other examples, will you add that in before we merge this?